devs2025.yml

```
@@ -28,19 +28,172 @@ translations:
+    t: |
+      Чим більше ви працюєте, тим більше заробляєте, принаймні до певної межі. Індія є найпрацьовитішою країною з медіанними 45 годинами роботи на тиждень, тоді як більшість інших країн зосереджені навколо позначки 40 годин.
+
+  - key: technology.workplace.job_happiness.takeaway.devs2025
+    t: |
+      Розробники загалом досить задоволені своєю роботою.
+
+  - key: technology.workplace.workplace_perks.takeaway.devs2025
+    t: |
+      Всупереч стереотипу асоціального комп'ютерного ботаніка, виявляється, що найбільше ми цінуємо насправді один одного — стосунки з колегами займають перше місце серед того, що розробники найбільше цінують у своєму робочому місці.
+
+  - key: technology.workplace.workplace_difficulties.takeaway.devs2025
+    t: |
+      Зрештою, всім нам потрібно заробляти на життя, і погана компенсація посіла перше місце як головна проблема робочого місця, з якою розробники стикаються сьогодні.
+
+  # technology
+
+  # - key: technology.mobile_os.takeaway.devs2025
+  #   t: |
+  #     xxx
+
+  # - key: technology.desktop_os.takeaway.devs2025
+  #   t: |
+  #     xxx
+
+  # - key: technology.programming_languages.takeaway.devs2025
+  #   t: |
+  #     xxx
+
+  - key: technology.productivity_apps.takeaway.devs2025
+    t: |
+      Коли йдеться про додатки для продуктивності, варто відзначити вражаючий результат **Obsidian**, який посів 6-те місце загалом, попри те, що навіть не був частиною попередньо визначених варіантів питання.
+
+      Деякі інструменти, такі як **Azure DevOps** та **Jira**, схильні до використання у великих компаніях, тоді як **Linear** та **Trello** популярніші серед менших організацій.
+
+      При розгляді виключно віддалених організацій,
+
+  - key: technology.chat_apps.takeaway.devs2025
+    t: |
+      Я настільки ж здивований, як і ви, дізнавшись, що **Google Chat** досі існує, проте він посів 9-те місце виключно завдяки відповідям, написаним від руки.
+
+  - key: technology.social_media.takeaway.devs2025
+    t: |
+      Mastodon та Bluesky можуть бути в авангарді нового контркультурного руху, але виявляється, що їхні користувачі також ті, кому потрібно розтягуватися щоразу, коли вони встають зі стільця, і які люблять розповідати про часи комутованого з'єднання та касетних стрічок.
+
+      TikTok, з іншого боку, схиляється до молодших користувачів, що має сенс, оскільки потрібне молоде тіло, щоб навіть *спробувати* деякі з тих танців.
+
+  - key: technology.community_involvement.takeaway.devs2025
+    t: |
+      Нічого дивного, але відео є основним способом, яким респонденти стежать за спільнотою. На жаль, особисті заходи залишаються досить рідкісними.
+
+      І хоча RSS-стрічки були популярною відповіддю, написаною від руки, медіанний вік цих респондентів не віщує добра для майбутнього цієї конкретної технології.
+
+  - key: technology.community_contributions.takeaway.devs2025
+    t: |
+      Приємно бачити, що розробники знаходять способи віддавати спільноті.
```

```suggestion
      Приємно бачити, що розробники знаходять способи віддячити спільноті.
```
тут треба якесь краще слово або словосполучення, але поки придумала тільки віддячити